flepimop/gempyor_pkg/tests/utils/test_flepimop_pull.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pytest
+from click.testing import CliRunner
+from unittest.mock import patch
+from gempyor.resume_pull import fetching_resume_files
```

Comment:
Is `tests/utils/test_flepimop_pull.py` the right place for this unit test? I would have a preference for placing this in `tests/resume_pull/test_flepimop_pull.py` so the unit tests are clearly broken up by module within `gempyor`.

Reply:
solve.